### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by KarmanjyotSingh**
Updated the experiment source code

### CODE REVIEW

File: experiment/simulation/assets/DragControls.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+  var _mouse = new THREE.Vector2();
+  var _offset = new THREE.Vector3();
+  var _intersection = new THREE.Vector3();
+
+  var _selected = null,
+    _hovered = null;
+
+  //
+
+  var scope = this;
```
Comment (raj-vlabs) [inline experiment/simulation/assets/DragControls.js:23]: Variables should not be declared using var, please use const or let.
Also variable names starting with underscores are used when the variables are private or carry some specific meaning. Why do we need to name them starting with an underscore?

File: experiment/simulation/assets/DragControls.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+  this.deactivate = deactivate;
+  this.dispose = dispose;
+
+  // Backward compatibility
+
+  this.setObjects = function () {
+    console.error("THREE.DragControls: setObjects() has been removed.");
+  };
+
+  this.on = function (type, listener) {
```
Comment (raj-vlabs) [inline experiment/simulation/assets/DragControls.js:210]: Why do we have deprecated functions here? Has this code been copied from somewhere?

File: experiment/simulation/index.html [hunk 3]
```diff
@@ -1,288 +1,252 @@
...
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/gh/virtual-labs/virtual-style@0.0.8-b/css/style.min.css"
+    />
+
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
+
+  <body class="page">
+    <section class="columns is-centered is-flex column">
```
Comment (raj-vlabs) [inline experiment/simulation/index.html]: please remove is-flex

File: experiment/simulation/index.html [hunk 4]
```diff
@@ -1,288 +1,252 @@
...
-                        <div class="v-table-wrap">
-                            <table id="table" class="table is-bordered is-fullwidth">
-                            </table>
-                        </div>
-                    </div>
+      </div>
+    </section>
+
+    <main
+      class="columns is-centered is-variable is-1-mobile is-flex is-flex-wrap-wrap workspace"
```
Comment (raj-vlabs) [inline experiment/simulation/index.html]: please remove is-flex

**Review by raj-vlabs — CHANGES_REQUESTED** [review]

File: experiment/simulation/assets/DragControls.js (hunk 1 above)
Comment (KarmanjyotSingh) [reply experiment/simulation/assets/DragControls.js:23]: This is the static file from the three.js official documentation

File: experiment/simulation/index.html (hunk 4 above)
Comment (KarmanjyotSingh) [reply experiment/simulation/index.html]: fixed

**Pull request merged**